module_utils/network_lsr/nm/client.py

```diff
@@ -4,24 +4,27 @@
 
 # Relative import is not support by ansible 2.8 yet
 # pylint: disable=import-error, no-name-in-module
-from ansible.module_utils.network_lsr.nm.error import LsrNetworkNmError  # noqa:E501
+from ansible.module_utils.network_lsr.nm import error  # noqa:E501
 
 import gi
 
-gi.require_version("NM", "1.0")
+try:
+    gi.require_version("NM", "1.0")
 
-# It is required to state the NM version before importing it
-# But this break the flake8 rule: https://www.flake8rules.com/rules/E402.html
-# Use NOQA: E402 to suppress it.
-from gi.repository import NM  # NOQA: E402
-from gi.repository import GLib  # NOQA: E402
-from gi.repository import Gio  # NOQA: E402
+    # It is required to state the NM version before importing it
+    # But this break the flake8 rule: https://www.flake8rules.com/rules/E402.html
+    # Use NOQA: E402 to suppress it.
+    from gi.repository import NM  # NOQA: E402
+    from gi.repository import GLib  # NOQA: E402
+    from gi.repository import Gio  # NOQA: E402
 
-# pylint: enable=import-error, no-name-in-module
+    # pylint: enable=import-error, no-name-in-module
 
-NM
-GLib
-Gio
+    NM
+    GLib
+    Gio
+except ValueError:
+    pass
```

Comment:
please add a comment here explaining that the `except ValueError; pass` is to workaround a bug in ansible 2.9 which causes this code to be executed on the control node (if that is the explanation)